brand_external_report_layout/views/report_template.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <template id="brand_external_layout" inherit_id="web.external_layout">
+        <xpath expr="//t[@t-if='not company']" position="after">
```

Comment:
Selecting on a `t-if` expression is fragile of course.

I wonder if this could be 
```suggestion
        <xpath expr="/" position="inside">
```
and then move `/*[@name!='...'` into the `else`.

Reply:
XPATH couldn't locate `/` in parent view.

```log
Element <xpath expr="/">' Element not found in parent view
```